v1/authentication_test.go

```diff
@@ -213,3 +214,50 @@ func TestContextualAuthenticatedIdentity_MiddlewareAndContext(t *testing.T) {
 		})
 	}
 }
+
+func TestAuthenticationIgnoresCertainEndpoints(t *testing.T) {
+	c := qt.New(t)
+
+	// Slice of endpoints that should not require authentication.
+	endpoints := []struct {
+		method string
+		path   string
+	}{
+		{
```

Comment:
just asking, but I imagine you have other parts in the code where you test auth is working.

Reply:
We test it in two points actually:
- One in the `core_test.go` does a full-handler test (from the HTTP server, down to the actual handler). This is the desired behavior we need to test; kind of like an integration test (between the components of the library).
- Another in the `authentication_test.go` which is more like a unit test for the authentication middleware, to verify that it doesn't invoke the user-populated authenticator middleware if it's `/swagger.json`.

Reply:
ty